pkg/netutil/cidr.go

```diff
@@ -3,9 +3,37 @@ package netutil
 import (
 	"net"
 
+	"github.com/stackrox/rox/pkg/logging"
 	"github.com/stackrox/rox/pkg/utils"
 )
 
+var (
+	privateSubnets = []*net.IPNet{
+		MustParseCIDR("127.0.0.0/8"),    // IPv4 localhost
```

Comment:
Note that this IP range (along with `::1`) is completely irrelevant to the problem at hand. I'm not even sure that a `127.0.0.0/8` CIDR block in a network policy has any effect at all.

Reply:
This is now removed as per the refactor I mentioned in the first comment. Check the latest commit: [0b1881a](https://github.com/stackrox/stackrox/pull/2929/commits/0b1881a15843dc5aca4d325b7454204a97efa327)